### ISSUE TITLE
[epic games free] add-on is no longer purchasing games

### ISSUE THREAD

**Issue by tijsrx** [alexbelgium/hassio-addons#767]
add-on version: 851cbbdc4d3e4e40d41f0358cefcdba281487758-2023-02-13 
since a couple of weeks the add-on shows the purchase page (where you have to click on confirm purchase) instead of the captcha page.

also, I've had captcha request since the very first purchase, is there any way to do anything about this on home-assistant? (please ELI5)

**Comment by alexbelgium** [comment]
Hi, please post on the upstream github - I only manage the addon https://github.com/claabs/epicgames-freegames-node
